package.json

```diff
@@ -46,6 +46,7 @@
     "babel-eslint": "5.0.1",
     "babel-loader": "5.3.3",
     "babel-plugin-rewire": "0.1.22",
+    "babel-polyfill": "^6.22.0",
```

Comment:
The generator functions require it, unfortunately. It works fine in Node, so we can remove it again when going to Electron.